ipfs-10-to-11/migration/migration.go

```diff
@@ -49,7 +49,18 @@ func (m Migration) Apply(opts migrate.Options) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	if err = transferPins(ctx, opts.Path); err != nil {
+	if !fsrepo.IsInitialized(opts.Path) {
+		return fmt.Errorf("ipfs repo %q not initialized", opts.Path)
+	}
```

Comment:
Why is this only used for Apply and not for Revert?

Reply:
Following example from previous migrations.

Done in both places now.